src/components/elements/MenuLink.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import MenuItem from 'material-ui/MenuItem'
+import ActionDelete from 'material-ui/svg-icons/action/delete'
+
+export const MenuLink = (type, id, onClick, dispatch) => {
+  validateType(type)
+  var typeTitle = type.charAt(0).toUpperCase() + type.slice(1)
```

Comment:
This will fail if the title is less than two characters. Also, may break on RTL or other non-English setups. Try using CSS: `text-transform: capitalize`

Reply:
Good point.  Deleted the capitalization.  And removed the validation fx per Maggie's request.  Validation is with PropTypes now.